README.md

```diff
@@ -38,34 +38,84 @@ The key/value pairs insides this ConfigMap are:
 
 #### Pointer to binding data
 
-This specification supports different scenarios for exposing bindable data.  
+This specification supports different scenarios for exposing bindable data. Below is a summary of [how to indicate what is interesting for binding](annotations.md).
```

Comment:
The new annotations.md doc is awesome!  I wonder if there's a way to make it more explicit that the summary below ls just a subset of what annotations.md covers.  Perhaps something like 

```
This specification supports different scenarios for exposing bindable data. Below is a summary of how to indicate what is interesting for binding.  Please see the [annotation section](annotations.md) for the full set with more details.
```

Reply:
Yes that could be cleared. Done in aa48f65